lib/template_handlers/markdown.rb

```diff
@@ -32,6 +32,8 @@ def initialize(template, source = nil, **options)
 
     def call
       # inspect objects to Ruby strings which can be eval'd
+      return %(#{render.inspect}.html_safe) if front_matter.empty?
```

Comment:
Can we stick in a quick unit test for this? for documenting why we're doing it as much as anything

Reply:
👍🏽